questions.js

```diff
@@ -0,0 +1,7 @@
+set up Jest with Babel or Webpack?
```

Comment:
you might want to put this file in your gitignore file

Reply:
oops lol ty